<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="9" passed="7">
  <reporter-output>
    <line>
      <![CDATA[F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205851-276_test005login.png]]>
    </line>
    <line>
      <![CDATA[<img src="../F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205851-276_test005login.png"/>]]>
    </line>
    <line>
      <![CDATA[F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205929-479_test000login.png]]>
    </line>
    <line>
      <![CDATA[<img src="../F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205929-479_test000login.png"/>]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="73559" started-at="2015-11-15T12:58:18Z" finished-at="2015-11-15T12:59:31Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="73559" started-at="2015-11-15T12:58:18Z" finished-at="2015-11-15T12:59:31Z">
      <class name="testTicketWeb.qiChe">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="setUp" is-config="true" duration-ms="2980" started-at="2015-11-15T20:58:18Z" finished-at="2015-11-15T20:58:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="test000login()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test000login" duration-ms="4264" started-at="2015-11-15T20:58:21Z" finished-at="2015-11-15T20:58:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test000login -->
        <test-method status="PASS" signature="test003login()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test003login" duration-ms="1" started-at="2015-11-15T20:58:25Z" finished-at="2015-11-15T20:58:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test003login -->
        <test-method status="PASS" signature="test004login()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test004login" duration-ms="1" started-at="2015-11-15T20:58:25Z" finished-at="2015-11-15T20:58:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test004login -->
        <test-method status="FAIL" signature="test005login()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test005login" duration-ms="22478" started-at="2015-11-15T20:58:25Z" finished-at="2015-11-15T20:58:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Can not find the µÇÂ¼ÊäÈë¿ò element on the µÇÂ¼Ò³ page.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Can not find the µÇÂ¼ÊäÈë¿ò element on the µÇÂ¼Ò³ page.
	at org.testng.Assert.fail(Assert.java:94)
	at core.UI.waitDisplay(UI.java:157)
	at core.UI.waitDisplay(UI.java:140)
	at initial.piaoWuWebTest.enterHomePage(piaoWuWebTest.java:13)
	at testTicketWeb.qiChe.test005login(qiChe.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205851-276_test005login.png]]>
            </line>
            <line>
              <![CDATA[<img src="../F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205851-276_test005login.png"/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- test005login -->
        <test-method status="PASS" signature="test006login()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test006login" duration-ms="1" started-at="2015-11-15T20:58:51Z" finished-at="2015-11-15T20:58:51Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test006login -->
        <test-method status="PASS" signature="test012editQiChePiaoJia()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="test012editQiChePiaoJia" duration-ms="23111" started-at="2015-11-15T20:58:51Z" finished-at="2015-11-15T20:59:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test012editQiChePiaoJia -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testTicketWeb.qiChe@34ce8af7]" name="tearDown" is-config="true" duration-ms="2313" started-at="2015-11-15T20:59:14Z" finished-at="2015-11-15T20:59:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testTicketWeb.qiChe -->
      <class name="testTicketWeb.baShi">
        <test-method status="PASS" signature="setUp()[pri:0, instance:testTicketWeb.baShi@59a6e353]" name="setUp" is-config="true" duration-ms="2488" started-at="2015-11-15T20:59:16Z" finished-at="2015-11-15T20:59:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="test000login()[pri:0, instance:testTicketWeb.baShi@59a6e353]" name="test000login" duration-ms="7064" started-at="2015-11-15T20:59:19Z" finished-at="2015-11-15T20:59:26Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [333334] but found [333]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [333334] but found [333]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at core.UI.assertEquals(UI.java:89)
	at testTicketWeb.baShi.test000login(baShi.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205929-479_test000login.png]]>
            </line>
            <line>
              <![CDATA[<img src="../F:\workspace\java\appautotest/failTestCaseScreenShot/20151115-205929-479_test000login.png"/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- test000login -->
        <test-method status="PASS" signature="test002login()[pri:0, instance:testTicketWeb.baShi@59a6e353]" name="test002login" duration-ms="121" started-at="2015-11-15T20:59:29Z" finished-at="2015-11-15T20:59:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test002login -->
        <test-method status="PASS" signature="test003login()[pri:0, instance:testTicketWeb.baShi@59a6e353]" name="test003login" duration-ms="0" started-at="2015-11-15T20:59:29Z" finished-at="2015-11-15T20:59:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test003login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testTicketWeb.baShi@59a6e353]" name="tearDown" is-config="true" duration-ms="2186" started-at="2015-11-15T20:59:29Z" finished-at="2015-11-15T20:59:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testTicketWeb.baShi -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
